pkg/views/gitprovider/select.go

```diff
@@ -17,11 +18,13 @@ import (
 )
 
 type GitProviderView struct {
-	Id         string
-	Name       string
-	Username   string
-	BaseApiUrl string
-	Token      string
+	Id            string
+	Name          string
+	Username      string
+	BaseApiUrl    string
+	Token         string
+	SigningMethod string
```

Comment:
You'll need to setup separate options there and then after the form runs pass the value of the SigningMethod input to the config if the value is not selected as None

P.S. PLease don't mark conversations as resolved yourself because it's easier for us to track